### PR TITLE
Feature/bar chart

### PR DESCRIPTION
**Pull request by WeiPeluso**
## Description

finished data visualization according to the bridge's status.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Organization

- [ ] Have you linked this pull request to a Trello card?

## Has This Been Tested

- [] Yes
- [x ] No
- [ ] Not necessary

## Review

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] Have you added two reviewers to this pull request?
- [ ] Is this a work in progress?
- [ ] Is this ready to be reviewed?
- [ ] My changes generate no new warnings
- [x ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by daniel-j-dev — APPROVED** [review]

File: src/styles/sass/bridgestatuschart.scss [hunk 1]
```diff
@@ -0,0 +1,50 @@
+.main {
+  //border: 2px solid red;
```
Comment (loganmetzger) [inline src/styles/sass/bridgestatuschart.scss:2]: Can probably delete these commented out borders

File: src/components/pages/DataViz/BridgeStatusChart.js [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
+        </a>
+      </Dropdown>
+      <div className="grid">{grid}</div>
+      <div className="chartContainer">
+        <div className="barChart">{barChar}</div>
+        <div className="doughnutChart">
+          <ReactEcharts option={option} />
+        </div>
+      </div>
+      <div></div>
```
Comment (BlueEagle) [inline src/components/pages/DataViz/BridgeStatusChart.js:342]: Does this need to exist?

**Review by BlueEagle — COMMENTED** [review]
Just working through an the PR review assignment, noticed this empty div.

File: src/components/pages/DataViz/BridgeStatusChart.js (hunk 2 above)
Comment (WeiPeluso) [reply src/components/pages/DataViz/BridgeStatusChart.js:342]: no, it was a mistake, we can delete it.

File: src/styles/sass/bridgestatuschart.scss (hunk 1 above)
Comment (WeiPeluso) [reply src/styles/sass/bridgestatuschart.scss:2]: Yes,we can.

**Pull request merged**